go/packages/dev-mode/main.go

```diff
@@ -145,28 +147,25 @@ func (e *Extension) ExternalExtension() {
 				requestId = agent.FindRequestId(arr)
 			}
 
+			hasInitErrorLog := agent.FindInitErrorLog(arr) == nil
```

Comment:
```suggestion
			hasInitErrorLog := agent.FindInitErrorLog(arr) != nil
```
shouldn't be `!=`?  🤔 when `== nil` then init error was not found

Reply:
Lol my bad I probably flipped this to do some tests and forgot to flip it back. Thanks for noticing 🙏